README.md

```diff
@@ -81,6 +81,11 @@ Manifests:
 }
 ```
 
+### Notes
+- For Shadow manager 2.1 synchronize configuration supports both data types for syncing shadows. The sync configuration
+can either be a `map` or a `list`. The map approach is preferred since this allows the customers to use the deployment 
+`merge` command to add in new shadows to sync.
+
```

Comment:
What would this look like if we are trying to remove a shadow from being synced?

Reply:
There is a [reset](https://docs.aws.amazon.com/greengrass/v2/developerguide/update-component-configurations.html#reset-configuration-update) command the customers can use during deployment.